pkg/apis/migration/v1alpha1/model.go

```diff
@@ -161,3 +167,81 @@ func GetSecret(client k8sclient.Client, ref *kapi.ObjectReference) (*kapi.Secret
 
 	return &object, err
 }
+
+// DeleteMigrated - dispatches delete requests for migrated resources
+func DeleteMigrated(config *rest.Config, uid string) error {
+	GVRs, err := getGVRs(config)
+	if err != nil {
+		return err
+	}
+	client, err := dynamic.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+	listOptions := k8sclient.MatchingLabels(map[string]string{
+		MigratedByLabel: uid,
+	}).AsListOptions()
+	foreground := metav1.DeletePropagationForeground
```

Comment:
This will ensure all of dependent resources will be removed (those which have `BlockOwnerDeletion` set on true) - https://github.com/konveyor/mig-controller/blob/e333d47d55dc6317e4a0495fafbb6cb991445204/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L420-L424